tests/Sapi/CliServer/CliServer.php

```diff
@@ -57,6 +57,8 @@ public function __construct($indexFile, $host, $port, array $envs = [], array $i
 
```

Comment:
All the changes done on this file is to prevent random errors due to cli server not stopping on time or not opening the socket on time